dali/pipeline/operators/util/external_source.cu

```diff
@@ -12,21 +12,54 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include <utility>
+#include <memory>
+
 #include "dali/pipeline/operators/util/external_source.h"
 
 namespace dali {
 
+template <>
+struct ExternalSource<GPUBackend>::RecycleFunctor {
+  RecycleFunctor() = default;
+  RecycleFunctor(const RecycleFunctor &) {
```

Comment:
so why not `= delete;`?

Reply:
std::function needs it somehow to exist.